src/modules/member/controllers/member.controller.ts

```diff
@@ -1,35 +1,158 @@
   ApiTags,
 } from '@nestjs/swagger';
-import { UserLogin } from '@src/decorators/user-login.decorator';
 import { ApiFailureResponse } from '@src/decorators/api-failure-response.decorator';
+import { SetModelNameToParam } from '@src/decorators/set-model-name-to-param.decorator';
+import { SetResponse } from '@src/decorators/set-response.decorator';
+import { UserLogin } from '@src/decorators/user-login.decorator';
+import { IdRequestParamDto } from '@src/dtos/id-request-param.dto';
+import { JwtAuthGuard } from '@src/guards/jwt-auth.guard';
+import { OptionalJwtAuthGuard } from '@src/guards/optional-auth-guard';
+import { UseDevelopmentInterceptor } from '@src/interceptors/use-development.interceptor';
+import { AuthService } from '@src/modules/core/auth/services/auth.service';
+import {
+  ApiFindOne,
+  ApiGetAccessTokenForDevelop,
+  ApiLoginOrSignUp,
+  ApiUpdateFromPatch,
+} from '@src/modules/member/controllers/member.swagger';
+import { PatchUpdateMemberRequestBodyDto } from '@src/modules/member/dtos/patch-update-member-request-body.dto';
+import { MemberValidationService } from '@src/modules/member/services/member-validation.service';
+import { AccessToken } from '@src/modules/member/types/member.type';
+import { InternalServerErrorResponseType } from '@src/types/internal-server-error-response.type';
+import { NotFoundResponseType } from '@src/types/not-found-response.type';
 import { LoginByOAuthDto } from '../dtos/create-member-by-oauth.dto';
 import { LastStepLoginDto } from '../dtos/last-step-login.dto';
+import { LoginOrSignUpRequestBodyDto } from '../dtos/login-or-sign-up-request-body.dto';
+import { MemberEntity } from '../entities/member.entity';
 import { MemberService } from '../services/member.service';
 import { MemberLastStepLoginResponseType } from '../types/response/member-last-step-login-response.type';
 import { MemberLoginByOAuthResponseType } from '../types/response/member-login-by-oauth-response.type';
-import { JwtAuthGuard } from '@src/guards/jwt-auth.guard';
 
-@ApiTags('멤버')
+/**
+ * @todo member 과제 통계 api 설명 한번 다시 듣고 구현
+ * @todo active member guard
+ *
+ */
+@ApiTags('멤버 (유저)')
+@ApiNotFoundResponse({ type: NotFoundResponseType })
+@ApiInternalServerErrorResponse({ type: InternalServerErrorResponseType })
 @Controller('api/members')
 export class MemberController {
-  constructor(private readonly memberService: MemberService) {}
+  constructor(
+    private readonly memberService: MemberService,
+    private readonly memberValidationService: MemberValidationService,
+    private readonly authService: AuthService,
+  ) {}
+
+  @ApiGetAccessTokenForDevelop('accessToken 발급 받기 (개발용)')
+  @UseInterceptors(UseDevelopmentInterceptor)
+  @Post('access-token/:id')
+  getAccessTokenForDevelop(@Param() params: { id: string }): string {
+    return this.authService.createAccessToken(+params.id);
+  }
+
+  @ApiFindOne('member 단일 조회')
+  @SetResponse('member')
+  @Get(':id')
+  findOne(
+    @SetModelNameToParam('member')
```

Comment:
```suggestion
    @SetModelNameToParam(ModelName.Member)
```

저희 enum.ts에 ModelName을 string 형태로 매핑해 놓은 enum이 있는데 해당 enum에 Member 추가해서 불러다가 사용하는 방식이 더 좋은 것 같습니다!